### ISSUE TITLE
Default orderstatusses are hidden

### ISSUE THREAD

**Issue by Goedkoopsteverf** [mollie/PrestaShop#220]
### Expected behavior
A working PSP plugin after several difficulties

### Actual behavior
The default order statussen 1 untill 10 from Prestashop are hidden, so again a faillure of a not working issue

![Screenshot_1](https://user-images.githubusercontent.com/71271481/95659587-505d0880-0b22-11eb-854f-45e5b5db313d.png)

### Steps to reproduce the behavior
After upgrading from V4.0.7 to V4.0.8 control the order statussettings within Prestashop and you will see
that most of the default statusses are hidden. This feature is not tested so it was said in message from the developer

![image](https://user-images.githubusercontent.com/71271481/95659666-e002b700-0b22-11eb-81a8-3ffe44a95cf3.png)

### Environment
* PrestaShop version: 1.7.6.5 
* Mollie module version: 4.0.9 
* PHP version: 7.2.34

**Comment by SimonasB88** [comment]
@Goedkoopsteverf hey, thanks for your insights, we have backloged this.
We will update you.

Best Regards.

**Comment by Goedkoopsteverf** [comment]
The reason that the default prestashop orderstatusses are hidden is that there is set a checkbox <NULL> in the databasestructure of ps_order_state by uncheck this item places by module_name the orderstatuss are again visible in the backoffice shopsettings

![Screenshot_1](https://user-images.githubusercontent.com/71271481/95850753-9946d400-0d51-11eb-9c61-e6b05ccac982.png)

**Comment by SimonasB88** [comment]
Tried to reproduce the issue with the same Prestashop version on the local machine but could not reproduce the client provided issue maybe there was some other modules that were installed or configured?

Steps how tried to reproduce: 

- Installed Prestashop 1.7.6.5
- Installed Mollie 4.0.7
- Updated Mollie module from 4.0.7 to 4.0.8 version.

Looked in the code we added how we hide old mollie statuses. The code sets “deleted” = 1 for mollie old statuses in the order_status table we do not modify the “module_name” column. 

Looking forward to hearing from you.
Best Regards.
